### PR TITLE
feat(badges): added completed count to serializer for badges

### PR DESCRIPTION
**Pull request by Her0elt**
Completed count for badges
Added a counter for how many users has completed/owns a badge
Link to Trello task: ...

Comments/issues:
idk if i like the name of the field, but could not come up with anything better than completed_count or just count

#145 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] PR name is descriptive
- [ ] The PR includes the link to the relevant Trello task
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

- [ ] `python manage.py runserver` was run locally witout errors

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ services:
     container_name: db
     restart: always
     ports:
-      - "3306:3306"
+      - "3305:3305"
```
Comment (olros) [inline docker-compose.yml]: 🤪

**Review by olros — CHANGES_REQUESTED** [review]
Usikker på om vi skal ha antall eller prosent som har en badge. Tror prosent hadde vært kulest

File: app/content/serializers/badge.py [hunk 2]
```diff
@@ -1,9 +1,15 @@
 from rest_framework import serializers
+from rest_framework.fields import SerializerMethodField
 
-from ..models import Badge
+from app.content.models import Badge, UserBadge
 
 
 class BadgeSerializer(serializers.ModelSerializer):
+    completed_count = SerializerMethodField()
```
Comment (Zenjjim) [inline app/content/serializers/badge.py:8]: number_of_completed_badges?

File: app/content/serializers/badge.py [hunk 3]
```diff
@@ -1,9 +1,24 @@
...
+            "id",
+            "title",
+            "description",
+            "badge_completion_percent",
+            "image",
+            "image_alt",
+        ]
+
+    def get_badge_completion_percent(self, obj):
+        user_amount = User.objects.all().count()
```
Comment (Zenjjim) [inline app/content/serializers/badge.py:22]: total_user_count

File: app/content/serializers/badge.py [hunk 4]
```diff
@@ -1,9 +1,24 @@
...
+        badge_completion_count = UserBadge.objects.filter(badge=obj).count()
+        return badge_completion_count / user_amount * 100
```
Comment (Zenjjim) [inline app/content/serializers/badge.py:24]: Tror det er bedre practice å sende det som desimal tall, men om @olros vil ha det som prosent fungerer det vel

File: docker-compose.yml (hunk 1 above)
Comment (Zenjjim) [reply docker-compose.yml:9]: Bruv

File: app/content/serializers/badge.py (hunk 4 above)
Comment (olros) [reply app/content/serializers/badge.py:24]: Dere kan bestemme her, akkurat det samme for meg

File: docker-compose.yml (hunk 1 above)
Comment (Her0elt) [reply docker-compose.yml:9]: lol mysql min er på 3306 orker ikke bytte den

**Comment by Eirsteir** [comment]
Hva med global_completion_percentage eller total_completion_percentage_by_all_users?

**Comment by olros** [comment]
> Et par forslag til: Hva med global_completion_percentage eller total_completion_percentage_by_all_users?

Stemmer for total_completion_percentage isåfall, må ikke bli for langt. Nærmer oss nesten maks-lengden på tweets 😄

**Comment by SveinJH** [comment]
> Et par forslag til: Hva med global_completion_percentage eller total_completion_percentage_by_all_users?

Likte best global_completion_percentage egentlig, men er ikke veldig nøye om det er global eller total synes jeg

**Comment by Eirsteir** [comment]
> > Et par forslag til: Hva med global_completion_percentage eller total_completion_percentage_by_all_users?
> 
> 
> 
> Stemmer for total_completion_percentage isåfall, må ikke bli for langt. Nærmer oss nesten maks-lengden på tweets 😄 

total_badge_completion_in_percentage_by_all_other_users_who_has_recieved_the_badge_in_the_tihlde_membership_system?

**Comment by SveinJH** [comment]
> > > Et par forslag til: Hva med global_completion_percentage eller total_completion_percentage_by_all_users?
> > 
> > 
> > Stemmer for total_completion_percentage isåfall, må ikke bli for langt. Nærmer oss nesten maks-lengden på tweets 😄
> 
> total_badge_completion_in_percentage_by_all_other_users_who_has_recieved_the_badge_in_the_tihlde_membership_system?

Den er fin! 🥇

**Review by Eirsteir — APPROVED** [review]
Well done 😄

**Review by olros — APPROVED** [review]

**Pull request merged**